librz/util/vector.c

```diff
@@ -471,6 +471,26 @@ RZ_API bool rz_pvector_join(RZ_NONNULL RzPVector *pvec1, RZ_NONNULL RzPVector *p
 	return true;
 }
 
+/**
+ * \brief Assign the pointer \p ptr at \p index in the pvector.
+ *
+ * \param vec The pvector to assign to.
+ * \param index The index to assign the pointer to.
+ * \param ptr The pointer to assign.
+ *
+ * \return The pointer stored at \p index before. Or NULL in case of failure.
+ */
+RZ_API void *rz_pvector_assign_at(RZ_BORROW RZ_NONNULL RzPVector *vec, size_t index, RZ_OWN RZ_NONNULL void *ptr) {
```

Comment:
why you need this when `rz_pvector_set` is available?

Reply:
`rz_pvector_set` is private. My thinking was, that it is better to have an equivalent function to the normal `rz_vector_assign_at`. So we don't mix up names with function between `RzVector` and `RzPVector`.

Reply:
well, it's an inline within the headers.

Reply:
Something is wrong with this week -.-
Yeah, it is not necessary to add it than. `rz_pvector_get` and `rz_pvector_set` is good enough. Let me close this one.